examples/azure-vm-simple-linux-managed-disk/README.md

```diff
@@ -0,0 +1,24 @@
+# Very simple deployment of a Linux VM
+
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-vm-simple-linux%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template allows you to deploy a simple Linux VM using a few different options for the Ubuntu version, using the latest patched version. This will deploy a A1 size VM in the resource group location and return the FQDN of the VM.
```

Comment:
[Based on this](https://github.com/hashicorp/terraform/pull/14115/files#diff-3ccea4997df6993ff3a98e3548cf9174R45) this will spin up an A0 - can we update this to match?